test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -728,6 +779,46 @@ public void CreateDisplayMetadata_IsFlagsEnum_ReflectsModelType(Type type, bool
         }
```

Comment:
Yeah, best to check `Name` and `Group` in that test. Might want to create a `EnumGroupAndNameComparer` if that needs to be done in more than one place.